spec/features/user_can_checkout_spec.rb

```diff
@@ -11,7 +11,18 @@
       add_item_to_cart
       visit cart_path(user_one)
       fill_in "days", with: 5
-      click_button "Checkout"
+      click_button "Pay with Card"
+
+      sleep(2)
+      stripe_iframe = all('iframe[name=stripe_checkout_app]').last
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.